src/build_log.cc

```diff
@@ -113,9 +113,9 @@ BuildLog::LogEntry::LogEntry(const string& output)
   : output(output) {}
 
 BuildLog::LogEntry::LogEntry(const string& output, uint64_t command_hash,
-  int start_time, int end_time, TimeStamp restat_mtime)
+  int start_time, int end_time, TimeStamp mtime)
```

Comment:
can you make this `const` so that one doesn't accidentally assign to it in the ctor now that it shares the name with the member variable?

Reply:
I could, but it follows the same pattern as `start_time` and `end_time` now, which are also the same as the member variables. Seems more consistent to leave this as is.

Reply:
fair enough